### PR TITLE
Add web activity for OpenMobile ACL (bug 1172510)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1172510

### CODE REVIEW

**Comment by ngokevin** [comment]
r+

File: package/iframe/main.js [hunk 1]
```diff
@@ -6,9 +6,38 @@
 
     // No trailing slash, please.
     var MKT_URL = '{domain}';
+/*
```
Comment (mstriemer) [inline package/iframe/main.js]: This should probably go away or have a comment describing why it's here.

File: package/iframe/main.js [hunk 2]
```diff
@@ -257,9 +305,34 @@
...
+                // only "marketplace-openmobile-acl", keep the request around,
+                // generating an unique id. When we receive back a message
+                // saying an activity is done, if the id matches one we have,
+                // post the result back to the activity caller).
+                msg.id = getRandomInt(0, Number.MAX_SAFE_INTEGER);
+                activitiesInProgress[msg.id] = req;
+                log('This activity needs to return, generated id: ', msg.id);
+                // 'marketplace-openmobile-acl' also needs to wait on a
+                // getFeature() promise before sending the activity message.
+                if (typeof navigator.getFeature !== 'undefined') {
```
Comment (mstriemer) [inline package/iframe/main.js]: Should we call `navigator.hasFeature('navigator.getFeature')` instead?


`</troll>`

**Comment by mstriemer** [comment]
Looks good to me

**Pull request merged**